documentation/modules/proc-create-infraconfig-custom-resource-olm-ui.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring-olm.adoc
+// rhassemblies/assembly-configuring-olm-rh.adoc
+
+[id="proc-create-infraconfig-custom-resource-olm-ui-{context}"]
+= Creating an infrastructure configuration custom resource using the {PlatformName} console
+
+You must create an infrastructure configuration custom resource to use {ProductName}. This example uses `StandardInfraConfig` for a standard address space.
+
+.Procedure
+
+. From the dropdown menu, select the `openshift-operators` project.
+
+. Click *Catalog > Installed Operators*.
+
+. Click *Add > Import YAML*. The Import YAML window opens.
+
+. Copy the following code for `StandardInfraConfig`:
++
+[source,yaml,options="nowrap",subs="attributes"]
+----
+apiVersion: admin.enmasse.io/v1beta1
+kind: StandardInfraConfig
+metadata:
+  name: default
+----
+
+. In the Import YAML window, paste the copied code and click *Create*. The StandardInfraConfig Overview page is displayed.
+
+. Click *Workloads > Pods*. In the *Readiness* column, the Pod status is `Ready` when the custom resource has been deployed.
+
+.Next steps
+
+* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#proc-create-auth-service-custom-resource-olm-ui-messaging[Create an authentication service custom resource using the {PlatformName} console]
```

Comment:
Right now the order of flow is infra config -> authentication service -> address space plan -> address plan. Instead, I think it should either be:

* authentication service -> infra config -> address space plan -> address plan